src/test/moment/create.js

```diff
@@ -568,6 +568,8 @@ test('parsing iso', function (assert) {
     minute = pad(4 + minOffset),
 
     formats = [
+        ['2011',                          '2011-01-01T00:00:00.000' + tz],
```

Comment:
Actually, this should be `2011-03-03T00:00:00.000`.

Per https://en.wikipedia.org/wiki/ISO_8601#Week_dates:
>  [YYYY] indicates the ISO week-numbering year which is slightly different from the traditional Gregorian calendar year

Reply:
(More details are provided in the link provided, #4410 is related)